src/options.ts

```diff
@@ -11,6 +12,8 @@ export interface TerminalScreenshotOptions {
   readonly backgroundColor: string;
   // Type of the screenshot to be generated. (default: png)
   readonly type: "png" | "jpeg";
+  // Name of the theme to use. (default: "")
+  readonly theme: ITheme; // values https://github.com/ysk2014/xterm-theme/blob/master/src/index.js
```

Comment:
Given the breaking change, I suggest adding a `BREAKING CHANGE` snippet to the PR, to explain the API/CLI additions/removals:

https://www.npmjs.com/package/semantic-release